File: cap07/src/algoritmos/Fibonacci.py
def fib(lim):
    serie = [0, 1]
    i = 2
    alcanzado = False 
    while not alcanzado:
        aux = serie[i-2] + serie[i-1] 
        alcanzado = aux > lim
        if not alcanzado:
            serie.append(aux)
        i = i + 1 
    return serie
 
print(fib(1000))

File: cap07/src/algoritmos/Fibonacci.js
function fib(lim) {
	let serie = [0, 1]; 
	let i = 2;
	let alcanzado = false; 
	while (!alcanzado) {
		let aux = serie[i-2] + serie[i-1];
		alcanzado = aux > lim;
		if (!alcanzado) {
			serie[i] = aux; 
		}
		i = i + 1; 
	}
	return serie; 
}
document.write(fib(1000));